docs/03-design/LZ-03-03.adoc

```diff
@@ -5,7 +5,7 @@
 Softwarearchitekt:innen können grundlegende Vorgehensweisen der Architekturentwicklung benennen, erklären und anwenden, beispielsweise:
 
 * Top-down- und Bottom-up-Vorgehen beim Entwurf (R1)
-* Sichtenbasierte Architekturentwicklung (R1)
+* Sichtenbasierte Architekturentwicklung, siehe <<rozanskiwoods>>, <<kruchten95>> (R1)
```

Comment:
These changes are also part of PR # 463:

You added 3 book references but the 3rd is missing here: starkeetalarc42
Do we need all the books referenced from somewhere in the learning goals?

Reply:
[starkeetalarc42] is just a collection of examples for complete architectures with arc42, no specific stuff on architectural views is explained. Therefore I consider it not appropriate for this sub-learning-goal.

Your question: "Do we need these references?": The original issue #422 called for exactly that. I personally consider references to be a useful and optional add-on to the curriculum and would like even more references to be used in other LGs too.